pkg/config/aliases.go

```diff
@@ -29,8 +29,6 @@ type (
 	Writer = model.Writer
 	// ReaderWriter is alias to model.ReaderWriter
 	ReaderWriter = model.ReaderWriter
-	// Loader is alias to model.Loader
-	Loader = model.Loader
```

Comment:
Is it safe to remove the alias without replacing with the new one?

Reply:
Nothing is using the aliased interface, I checked.